app/components/post_body/post_body.js

```diff
@@ -13,6 +13,7 @@ import Icon from 'react-native-vector-icons/Ionicons';
 import {Posts} from 'mattermost-redux/constants';
 
 import CombinedSystemMessage from 'app/components/combined_system_message';
+import {renderSystemMessage} from './system_message_helpers';
```

Comment:
consider moving this file to its own component folder or as an utility in the utils folder, just so it is easier to find and maintain

Reply:
Since this is only used by `PostBody`, I'd actually prefer to leave it here. I don't think we're going to reuse `renderSystemMessage` elsewhere, so I don't think it should go into `utils`

Reply:
I considered moving this helper to a more generic location, but decided to follow the webapp here too. (see https://github.com/mattermost/mattermost-webapp/blob/master/components/post_markdown/post_markdown.jsx#L9). Perhaps this can move to `utils`, once these system message renderings get more consolidated and used universally?